### PR TITLE
add populator manager

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: Elastica/Populator/Strategies/ContentPopulator.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    protected $languages;
+    protected $multipleIndexor;
+    protected $contentRepository;
+
+    /**
+     * @param MultipleDocumentIndexorInterface $multipleIndexor
+     * @param ContentRepositoryInterface       $contentRepository
+     * @param array                            $languages
+     */
+    public function __construct(MultipleDocumentIndexorInterface $multipleIndexor, ContentRepositoryInterface $contentRepository, $languages)
```
Comment (ngilain) [inline Elastica/Populator/Strategies/ContentPopulator.php]: you can type $languages as an array in the signature

**Comment by nicolasThal** [comment]
@itkg-ngilain pr updated

**Comment by glenouvel** [comment]
:+1:

**Comment by ngilain** [comment]
:+1:
Do you have a doc on these features ?

**Comment by nicolasThal** [comment]
@itkg-ngilain I haven't written the doc yet for this part. I will do it next week probably

**Pull request merged**